doc/src/conf.py

```diff
@@ -30,6 +30,14 @@
               'sphinx.ext.mathjax', 'numpydoc', 'sympylive',
               'sphinx.ext.graphviz', 'matplotlib.sphinxext.plot_directive']
 
+redirects = {
+    "install.rst": "getting_started/install.rst",
+    "aboutus.rst": "getting_started/aboutus.rst",
+    "wiki.rst": "getting_started/wiki.rst",
```

Comment:
The wiki page is quite brief https://docs.sympy.org/dev/wiki.html and if the wiki houses the development docs, then maybe we only need a link pointing developers to the wiki. This page could be deleted.